juniper_official/Linecard/aft-server-entry-stats.rule

```diff
@@ -0,0 +1,41 @@
+healthbot {
+    topic external {
+        rule aft_server_entry_stats {
```

Comment:
it would be good if you can add comments on where this rule is being used. Which serves as reference in future